Build warnings in LastValueAggregatorTest

There are 4 build warnings regarding unassigned variables in LastValueAggregatorTest

```
Impl/Metrics/Aggregators/LastValueAggregatorTest.cs(28,37): warning CS0649: Field 'LastValueAggregatorTest.UpdateThreadArguments<T>.mreToBlockUpdateThread' is never assigned to, and will always have its default value null [/Users/mgoldsmith/code/opentelemetry-dotnet/test/OpenTelemetry.Tests/OpenTelemetry.Tests.csproj]
Impl/Metrics/Aggregators/LastValueAggregatorTest.cs(31,43): warning CS0649: Field 'LastValueAggregatorTest.UpdateThreadArguments<T>.LastValueAggregator' is never assigned to, and will always have its default value null [/Users/mgoldsmith/code/opentelemetry-dotnet/test/OpenTelemetry.Tests/OpenTelemetry.Tests.csproj]
Impl/Metrics/Aggregators/LastValueAggregatorTest.cs(30,24): warning CS0649: Field 'LastValueAggregatorTest.UpdateThreadArguments<T>.threadsStartedCount' is never assigned to, and will always have its default value 0 [/Users/mgoldsmith/code/opentelemetry-dotnet/test/OpenTelemetry.Tests/OpenTelemetry.Tests.csproj]
Impl/Metrics/Aggregators/LastValueAggregatorTest.cs(29,37): warning CS0649: Field 'LastValueAggregatorTest.UpdateThreadArguments<T>.mreToEnsureAllThreadsStart' is never assigned to, and will always have its default value null [/Users/mgoldsmith/code/opentelemetry-dotnet/test/OpenTelemetry.Tests/OpenTelemetry.Tests.csproj]
```

--- a/test/OpenTelemetry.Tests/Impl/Metrics/Aggregators/LastValueAggregatorTest.cs
+++ b/test/OpenTelemetry.Tests/Impl/Metrics/Aggregators/LastValueAggregatorTest.cs
@@ -23,18 +23,10 @@ namespace OpenTelemetry.Metrics.Test
 {
     public class LastValueAggregatorTest
     {
-        private class UpdateThreadArguments<T> where T: struct
-        {
-            public ManualResetEvent mreToBlockUpdateThread;
-            public ManualResetEvent mreToEnsureAllThreadsStart;
-            public int threadsStartedCount;
-            public LastValueAggregator<T> LastValueAggregator;
-        }
-
         [Fact]
         public void LastValueAggregatorSupportsLong()
         {
-            LastValueAggregator<long> aggregator = new LastValueAggregator<long>();            
+            LastValueAggregator<long> aggregator = new LastValueAggregator<long>();
         }
 
         [Fact]
@@ -46,7 +38,7 @@ namespace OpenTelemetry.Metrics.Test
         [Fact]
         public void LastValueAggregatorConstructorThrowsForUnSupportedTypeInt()
         {
-            Assert.Throws<Exception>(() => new LastValueAggregator<int>());            
+            Assert.Throws<Exception>(() => new LastValueAggregator<int>());
         }
 
         [Fact]

